samples/defer3.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+)
+
+func f1() {
+	for i := 0; i < 3; i++ {
+		defer fmt.Print(i, " ")
+	}
+}
+
+func f2() {
+	for i := 0; i < 3; i++ {
+		defer func() {
+			fmt.Print(i, " ")
+		}()
+	}
+}
+
+// returns 42
+func f3() (result int) {
+
```

Comment:
Remove extra line to be consistent with the other functions

Reply:
Done